server/router.js

```diff
@@ -152,10 +152,10 @@ Input:
   req.params.userId: String user ID to be added to the group.
 Output: Returns json file with the updated group information.
 */
-router.post('/api/group/:groupId/user/:userId', (req, res) => {
+router.post('/api/group/:groupId/user', (req, res) => {
   const groupId = req.params.groupId;
-  const userId = req.params.userId;
-  if (req.isAuthenticated() && req.user.isMemberOf(groupId)) {
+  const userId = (req.query.userId) ? req.query.userId : req.user.id;
```

Comment:
how's this look everyone to address the things I mentioned above? @ang-zhu @ethanbb